.tx/integration_config.yml

```diff
@@ -0,0 +1,28 @@
+settings:
+  language_mapping:
```

Comment:
I guess we don't need the mapping and the `release_notes` file once we have two transifex projects? Not a blocker for now though

Reply:
💯 , left it to complete the full setup